src/query/storage/interface.go

```diff
@@ -86,11 +87,31 @@ type FetchOptions struct {
 type Querier interface {
 	// Fetch fetches timeseries data based on a query
 	Fetch(
-		ctx context.Context, query *FetchQuery, options *FetchOptions) (*FetchResult, error)
-	FetchTags(
-		ctx context.Context, query *FetchQuery, options *FetchOptions) (*SearchResults, error)
+		ctx context.Context,
+		query *FetchQuery,
+		options *FetchOptions,
+	) (*FetchResult, error)
+
+	// FetchRaw returns the raw iterators from the underlying storage
+	FetchRaw(
```

Comment:
We shouldn't expose this as part of Querier. This couples us very strongly with dbnode. Shouldn't the local storage FetchBlock internally use FetchRaw ? Not sure why Engine needs to know about this.